DotNetSample/DataIndexer/DataIndexer.csproj

```diff
@@ -44,27 +44,27 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Search, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.Azure.Search.5.0.0\lib\net452\Microsoft.Azure.Search.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Search, Version=6.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Azure.Search.6.0.0-preview\lib\net452\Microsoft.Azure.Search.dll</HintPath>
```

Comment:
If the goal here was to use the latest, you should use 8.0.0-preview, not 6.0.0-preview.

Reply:
This was not intentionally part of this PR.  Can I remove the file?

Reply:
Yeah, go for it.